content/en-us/reference/engine/classes/TextChannel.yaml

```diff
@@ -159,19 +159,20 @@ callbacks:
       be implemented on the client.
 
       Use this to decorate `Class.TextChatMessage|TextChatMessages`. If this
-      callback returns a `Class.TextChatMessageProperties`, those properties are
-      merged with the `Class.TextChatMessage` parameter to create a new
-      `Class.TextChatMessage` with those properties.
+      callback returns a `Class.TextChatMessageProperties`, those properties 
+      are merged with the `Class.TextChatMessage` parameter to create a new
+      `Class.TextChatMessage`.
 
-      When bound to the client sending a message, this event can be fired twice,
-      first when the message is initially sent and received by the same client
-      locally, and then when the client receives the result of the filtered
-      message from the server. `Class.TextChannel.OnIncomingMessage` callbacks
-      always run **after** `Class.TextChatService.OnIncomingMessage` callback.
+      When bound to the client sending a message, this callback is run twice;
+      first when the message is initially sent and received locally, and again when the client receives the result of the filtered message from the server.
+      
+      `Class.TextChannel.OnIncomingMessage` callbacks always run **after** the `Class.TextChatService.OnIncomingMessage` callback.
 
       This should be defined only once per `Class.TextChannel` in the source
       code. Multiple bindings to the same channel will override one another in a
       nondeterministic manner.
+
+      When `Class.TextChatService.CreateDefaultTextChannels` is true, those default `Class.TextChannel|TextChannels` have their `Class.TextChannel.OnIncomingMessage` callbacks assigned internally in order to [exhibit special default behavior.](../reference/engine/classes/TextChatService#CreateDefaultTextChannels)
```

Comment:
```suggestion
      When `Class.TextChatService:CreateDefaultTextChannels()` is true,
      those default `Class.TextChannel|TextChannels` have their
      `Class.TextChannel.OnIncomingMessage` callbacks assigned
      internally in order to exhibit special default behavior.
```

Reply:
Is this the correct way to make hyperlinks to API pages?
> `` [exhibit special default behavior.](`Class.TextChatService.CreateDefaultTextChannels`) ``

In most cases I was just using this syntax: `` `Class.TextChannel` ``
But if I want to take some existing words and have it link to an API page, I'm not sure if this would continue working.